ft/persistence-modbus-tcp/persistence-simple-1k/adapters.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Why remove the data sub directory? Please keep it there, and replace directory `persistence-modbus-tcp` to `data`